### PR TITLE
Refactor to use new interactives api model

### PR DESCRIPTION
**Pull request by mryan321**
### What

- use `UploadRootDirectory` from API response, instead of searching for file within archive files collection - not efficient in case of 000s files. If it cant find file downstream will just 404.
- some housekeeping from renaming: https://github.com/ONSdigital/dp-api-clients-go/pull/287

### How to review

👀 

### Who can review

Anyone

### CODE REVIEW

File: go.mod
Comment (gilesburdett) [inline go.mod]: Mistakenly left in during development / waiting for new release of dp-api-clients-go?

Comment (mryan321) [reply go.mod]: Yep - waiting on https://github.com/ONSdigital/dp-api-clients-go/pull/287 - once merged will bump go.mod and drop replace 👍😎

**Review by greg-fujar — APPROVED** [review]

**Pull request merged**
